Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/AttachmentReadback.h

```diff
@@ -25,8 +25,11 @@ namespace AZ
 
     namespace RPI
     {
-        //! A scope producer which reads back buffer or image attachments 
-        class AttachmentReadback
+        //! A scope producer which reads back one or more attachments to CPU memory.
```

Comment:
Done. Changed to:  
```
        //! Adds one or two scope producers ((the second is to decompose an MS Texture)
        //! which read back one or more attachments to CPU memory.
        //! Both buffer and image attachments are supported.
        //! In case of images it can also capture specific Mip Levels, as defined in the image view descriptors.
        //! Also, for images, Volume Texture image attachments (aka Texture3D) are supported too.
```